ng-sortable.js

```diff
@@ -46,16 +46,16 @@
 
 					var ngRepeat = [].filter.call($element[0].childNodes, function (node) {
 						return (
-								(node.nodeType === 1) &&
-								(node.attributes['ng-repeat'])
-							);
+							node.nodeType === Node.ELEMENT_NODE &&
+							angular.element(node).data('ngRepeat')
+						);
 					})[0];
 
 					if (!ngRepeat) {
 						return;
 					}
 
-					var expression = ngRepeat.attributes['ng-repeat'].nodeValue;
+					var expression = angular.element(ngRepeat).data('ngRepeat');
```

Comment:
`angular.element(ngRepeat).data` — it works with only data-attributes.
 
Correct:
```js
function getRepeatValue(el) {
    return ngRepeat.getAttribute('ng-repeat') || ngRepeat.getAttribute('data-ng-repeat') ||  || ngRepeat.getAttribute('x-ng-repeat');
}
```